includes/class-wp-document-revisions-admin.php

```diff
@@ -610,7 +613,7 @@ public function revision_metabox( $post ) {
 			}
 			?>
 			<tr>
-				<td><a href="<?php echo esc_url( $fn ); ?>" title="<?php echo esc_attr( $revision->post_modified ); ?>" class="timestamp" id="<?php echo esc_attr( strtotime( $revision->post_modified ) ); ?>"><?php echo esc_html( human_time_diff( strtotime( $revision->post_modified_gmt ), time() ) ); ?></a></td>
```

Comment:
Curious the motivation for this change here?

Reply:
In fact, I actually then thought of the id itself - and thought what is actually its purpose/usefulness (and could the document and its latest revision have the same id.
So I removed it in the revision metabox.

I should have removed the id attribute from the document metabox as well.

Happy to do that.
Neil